web/src/actions/print.js

```diff
@@ -0,0 +1,6 @@
+export const PRINT_APD = Symbol('print : apd');
+
+export const printApd = ({ global = window } = {}) => dispatch => {
+  dispatch({ type: PRINT_APD });
```

Comment:
Later, we can have a reducer that responds to this action to update some piece of state, and we can use that piece of state to trigger UI changes elsewhere?  I was thinking, for example, that our `Input` components could check that state and switch from inputs to just text blocks.  That'd be pretty neat.